Kickstarter-iOS/Views/Cells/DiscoveryEditorialCell.swift

```diff
@@ -109,15 +117,17 @@ final class DiscoveryEditorialCell: UITableViewCell, ValueCell {
   // MARK: - Configuration
 
   private func configureViews() {
-    _ = (self.containerView, self.contentView)
+    _ = (self.editorialImageView, self.contentView)
       |> ksr_addSubviewToParent()
       |> ksr_constrainViewToMarginsInParent()
 
-    _ = (self.rootStackView, self.containerView)
+    _ = (self.containerView, self.editorialImageView)
```

Comment:
We shouldn't make regular `UIView`s subviews of `UIImageView`s. In this case you can just add `editorialImageView` to `contentView` and the `containerView` above it both constrained to the edges of `contentView`.